.github/renovate.json

```diff
@@ -22,9 +22,10 @@
       "description": "We focus on Guice 6 until core adapts 7"
     },
     {
-      "matchPackagePatterns": [
-        "selenium"
-      ],
+      "matchPackageNames": ["gitlab/gitlab-ce"]
```

Comment:
this will be hidden by `ignorePaths` on line 9, dependabot appears to be used for docker updates:
https://github.com/jenkinsci/acceptance-test-harness/blob/master/.github/dependabot.yml

it would probably be better to remove all the dependabot docker config though and just switch it to renovate as then we wouldn't need to explicitly list it

Reply:
@ampuscas This renovate bit can be reverted now - the plan is to use dependabot for this PR and then make another PR to deal with switching from Dependabot to Renovate 🙂